tests/integration/test_multus_v3_8.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2024 Canonical, Ltd.
+#
+import os
+import pathlib
+
+from test_util import harness, util
+
+
+def test_multus_deployment(
+        tmp_path: pathlib.Path,
+        module_instance: harness.Instance):
+    clone_path = tmp_path / "multus"
+    clone_path.mkdir()
+
+    clone_command = [
+        "git", "clone", "https://github.com/k8snetworkplumbingwg/helm-charts",
+        "--depth", "1",
+        str(clone_path.absolute())
+    ]
+    module_instance.exec(clone_command)
+
+    chart_path = clone_path / 'multus'
+
+    helm_command = [
+        "sudo", "k8s",
+        "helm", "install", "multus-cni",
+        str(chart_path.absolute()),
+        "--namespace", "kube-system",
+    ]
+
+    image_uri = os.getenv("ROCK_MULTUS_V3_8")
+    assert image_uri is not None, "ROCK_MULTUS_V3_8 is not set"
```

Comment:
might want to have this assert earlier. No point in doing the above if this doesn't exist.

Reply:
Good point. This was merged already but I can add another commit here: https://github.com/canonical/multus-rock/pull/7